spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -25,6 +25,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
```

Comment:
Is it really so mandatory dependency ?

Why do we need it at all?
Is there some comment in the commit message explaining such a change?
Thanks